src/generate_text.py

```diff
@@ -241,6 +261,11 @@ def main(flags):
                         type=int,
                         default=10
                         )
+    parser.add_argument('--device',
```

Comment:
please add a choice here between cpu and gpu

Reply:
don't get it.